dev/com.ibm.ws.sipcontainer/src/com/ibm/ws/sip/container/util/SipLogExtension.java

```diff
@@ -188,6 +190,9 @@ private static String getCallId() {
 			}
 		}
 		
+    	} catch (NullPointerException e ) {
+    		
+    	}
```

Comment:
Can you add a comment to explain the npe catches one from the logging?

Reply:
Comments added